ui/app/components/secret-delete-menu.js

```diff
@@ -24,92 +22,21 @@ export default class SecretDeleteMenu extends Component {
 
   @tracked showDeleteModal = false;
 
-  @maybeQueryRecord(
```

Comment:
When navigating from the details page back to the list, the console was erroring with 
```Cannot create a new tag for `<model::capabilities:undefined>` after it has been destroyed```
Finally tracked it down to these not tearing down gracefully, so I moved the capabilities to the model instead which is our latest pattern

Reply:
It looks like the `secret-edit` component also uses `maybeQueryRecord` so something to be aware of if we notice that error further. Interesting that it's also used in `lazy-capabilities` which is used a bunch but maybe it's ok when used on a model since they aren't torn down as frequently as components.